src/main/java/net/dv8tion/jda/api/Permission.java

```diff
@@ -45,6 +45,7 @@
     BAN_MEMBERS(          2, true, false, "Ban Members"),
     NICKNAME_CHANGE(     26, true, false, "Change Nickname"),
     NICKNAME_MANAGE(     27, true, false, "Manage Nicknames"),
+    MODERATE_MEMBERS(    40, true, false, "Moderate Members"),
```

Comment:
This is "Timeout Members" now
```suggestion
    TIMEOUT_MEMBERS(     40, true, false, "Timeout Members"),
```
![image](https://user-images.githubusercontent.com/8278263/146809044-e11f3ab9-601c-4615-904e-46affd430470.png)